manager/controllers/app/select_modules.go

```diff
@@ -143,13 +144,13 @@ func (p *PlotterGenerator) validate(item *DataInfo, solution Solution, appContex
 		requiredActions = unsupported
 		// select a cluster for the capability that satisfy cluster restrictions specified in admin config policies
 		if !p.findCluster(item, element, appContext) {
-			p.Log.V(0).Info("Could not find an available cluster for " + moduleCapability.Capability)
+			p.Log.Error().Str(logging.DATASETID, item.Context.DataSetID).Msg("Could not find an available cluster for " + moduleCapability.Capability)
 			return false
 		}
 	}
 	// Are all actions supported by the capabilities in this data path?
 	if len(requiredActions) > 0 {
-		p.Log.V(0).Info("Not all governance actions are supported, aborting data path construction")
+		p.Log.Fatal().Str(logging.DATASETID, item.Context.DataSetID).Msg("Not all governance actions are supported, aborting data path construction")
```

Comment:
Debug, not Fatal. Path is invalidated if not all actions can be performed along the path.